lib/mp4/transmuxer.js

```diff
@@ -822,7 +834,8 @@ CoalesceStream.prototype.flush = function(flushSource) {
     offset = 0,
     event = {
       captions: [],
-      metadata: []
+      metadata: [],
+      info: {}
```

Comment:
I kind of wish we had a more descriptive name for this field but it's not worth holding up the PR on.